src/CMakeLists.txt

```diff
@@ -40,6 +40,18 @@ ADD_LIBRARY(${LIBRARY_NAME}
   command/command.cpp
   )
 
+INSTALL(TARGETS ${PROJECT_NAME}
+  EXPORT ${TARGETS_EXPORT_NAME}
+  PUBLIC_HEADER 
+  INCLUDES DESTINATION include/${PROJECT_NAME}
+  )
+
+target_include_directories(${PROJECT_NAME}
+  PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
```

Comment:
I think we should not need to add anything to get the headers of a package working in this package. If all headers are in the main `include/` dir, CMake should find them.

Reply:
Not if you want the include directory to be exported. It you want this, you must do something.